js/src/lib/Details/Usage.js

```diff
@@ -1,27 +1,49 @@
 import React from 'react';
 import { packageLink } from '../util';
 
-const Usage = ({ dependencies }) => (
+const Deps = ({ dependencies, text, id }) =>
+  dependencies
+    ? <div className="d-flex flex-items-between w-100">
+        <img src={`/assets/detail/ico-${id}.svg`} alt="" />
+        <dt>
+          {Object.keys(dependencies).length > 0
+            ? <details>
+                <summary>{text}</summary>
+                {Object.keys(dependencies)
```

Comment:
This is calling `Object.keys` there times, which will iterate over the object unnecessarily. Can you just call it once and cache it in a variable (like `const deps = Object.keys(dependencies)`?